src/nsls2api/api/v1/jobs_api.py

```diff
@@ -83,8 +83,9 @@ async def sync_proposal_types(facility: FacilityName = FacilityName.nsls2):
     include_in_schema=SYNC_ROUTES_IN_SCHEMA,
     tags=["sync"],
 )
-async def sync_proposals_for_cycle(request: Request, cycle: str) -> BackgroundJob:
-    sync_params = JobSyncParameters(cycle=cycle)
+async def sync_proposals_for_cycle(request: Request, cycle: str,
```

Comment:
where is the best place to the facility in the URL, if that is possible? or would this be part of the request instead?